lib/transforms/get-values.js

```diff
@@ -140,6 +141,9 @@ module.exports = {
           key = getNestedProperty(el, 'attributes.role.value') || getNestedProperty(el, 'attributes.role.type');
           value = getNestedProperty(el, 'attributes.summary_title') || null;
           link = getNestedProperty(el, 'links.self');
+          if (makersList.indexOf(key) === -1) {
+            key = 'maker';
```

Comment:
Just to clarify, if the type of the maker is not in the list we are defining the type "maker" by default, is that right?